message_ix_models/data/costs/eric-fom-costs.csv

```diff
@@ -0,0 +1,70 @@
+# Eric's adjusted/manual fixed O&M costs for MESSAGE technologies
+#
+# Units: 2005 USD per kW
+# 
+# - This is copied directly from the RegionDiff sheet in SSP1_techinput.xlsx

```

Comment:
Please give the complete path to this file, whether on the IIASA P-drive or elsewhere, so that someone could locate it.

Rather than "Eric's" in the title and file name, perhaps use:
- fix_cost-0.csv as a file name—"fix_cost" aligns with the name of the MESSAGEix parameter that is eventually targeted; "-0" indicates there might be multiple alternatives (Eric's, Alice's, Bob's, etc.)
- Add a bullet like "These were prepared ca. [DATE] by Eric Lastname [details…]."